registry.yaml

```
@@ -999,6 +999,24 @@
   path: examples/vector_databases/deeplake/deeplake_langchain_qa.ipynb
   date: 2023-09-30
   authors:
-    - FayazRahman
+    - ash0ts
```

was this intentional? 

```suggestion
    - FayazRahman
```